Makefile

```diff
@@ -38,7 +38,7 @@ test-force:
 	pytest -n auto --force-regen
 
 uv-test: test-data-gds
-	uv run pytest -s -n auto
+	uv run pytest -s -n logical
```

Comment:
This makes it possible for more cores to be used sometimes, it falls back to auto if it cant detect any logical CPU cores

Reply:
perfect, Thank you Matthew!